<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KronosClass</class>
 <widget class="QWidget" name="KronosClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1124</width>
    <height>947</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kronos</string>
  </property>
  <widget class="QPushButton" name="forwardbtn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Forward</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backbtn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Backward</string>
   </property>
  </widget>
  <widget class="QPushButton" name="leftbtn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Left</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rightbtn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Right</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connectbtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Output</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="disconnectbtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>DISCONNECTED</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Units</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="yvel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>160</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="xvel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>200</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>180</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>X Velocity</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>140</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Y Velocity</string>
   </property>
  </widget>
  <widget class="QPushButton" name="runbtn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopbtn">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="RenderWindow" name="emccdDisp">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>551</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="RenderWindow" name="cmosDisp">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>520</y>
     <width>531</width>
     <height>411</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>-13</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>500</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>EMCCD</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>500</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>CMOS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="toggleScaleBtn">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>490</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>100x</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmosMoveBtn">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>490</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Move</string>
   </property>
  </widget>
  <widget class="QPushButton" name="acquireBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>490</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Acquire</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>RenderWindow</class>
   <extends>QOpenGLWidget</extends>
   <header>renderwindow.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="kronos.qrc"/>
 </resources>
 <connections/>
</ui>
